src/sidebar/index.js

```diff
@@ -91,9 +86,9 @@ function autosave(autosaveService) {
 }
 
 // @inject
-function setupFrameSync(frameSync) {
+function setupFrameSync(frameSync, isSidebar) {
   if (isSidebar) {
-    frameSync.connect();
+    frameSync.connect(true);
```

Comment:
Since this code doesn't run until after the route is determined by calling `router.sync()` in the `setupRoute` function, you could read the route ID from the store at this point (`store.route() === 'sidebar'`) rather than using the `isSidebar` variable. This avoids having two places in the code that map a URL to a route name (the router service and the code that initializes `isSidebar` here)

Reply:
I kind of like the layer of abstraction that `isSidebar` gives us. That change would couple the logic to the route.

Reply:
I'm not sure I follow. The initial value of `isSidebar` _should_ be equivalent to `store.route() === "sidebar"` after the route is initialized for the first time.